services/QuillLMS/app/controllers/api/v1/rule_feedback_histories_controller.rb

```diff
@@ -24,4 +24,12 @@ def prompt_health
     report = PromptFeedbackHistory.run(**options)
     render json: report
   end
+
+  def activity_health
+    raise ArgumentError unless params.include?('activity_id')
+
+    options = params.permit(:activity_id).to_h.symbolize_keys
+    report = ActivityFeedbackHistory.run(**options)
```

Comment:
The arg should be passed in as `activity_id: <value>`, rather than `**options`, because `activity_id` is the only keyword arg that ActivityFeedbackHistory currently handles. To pass in `**options` suggests that there are other options to handle.

Line 31 can also be removed, so the final code is:

`report = ActivityFeedbackHistory.run(activity_id: params['activity_id'))`